models/HarDUNet.py

```diff
@@ -0,0 +1,186 @@
+import os
+import yaml
+import torch.nn as nn
+from models.helper import Bottleneck, Down, Up, Conv
+from models.config_dic import config_files
+
+
+class HarDUNet(nn.Module):
+    def __init__(
+        self,
+        n_classes=1,
+        arch="68",
+        act="relu",
+        transformer=False,
+        keepbase=False,
+        bilinear=True,
+        *args,
+        **kwargs,
+    ):
+        super().__init__()
+        # Down and Up U-Net
+        self.classes = n_classes
+        config_path = os.path.join(os.getcwd(), "models", "configs", config_files[arch])
+        with open(config_path, "r") as file:
+            config = yaml.safe_load(file)
+
+        second_kernel = 3
+        init_ch = 1
+        first_ch = config.get("first_ch")[0]
+        ch_list = config.get("ch_list")[0]
+        gr = config.get("gr")[0]
+        m = config.get("grmul")
+        n_layers = config.get("n_layers")[0]
+        downSamp = config.get("downSamp")[0]
```

Comment:
This variable is not used? I think the code was commented out below, can you check?